app/Providers/RepoServiceProvide.php

```diff
@@ -33,10 +32,10 @@ public function boot()
     public function register()
     {
         $this->app->bind(UserInterface::class, UserRepository::class);
-//       $this->app->bind(OrganizerInterface::class, OrganizerRepository::class);
-       $this->app->bind(OrganizerInterface::class, OrganizerAPIRepo::class);
-//       $this->app->bind(SportEventInterface::class, SportEventRepository::class);
-       $this->app->bind(SportEventInterface::class, SportEventAPIRepo::class);
+       $this->app->bind(OrganizerInterface::class, OrganizerRepository::class);
+//       $this->app->bind(OrganizerInterface::class, OrganizerAPIRepo::class);
+       $this->app->bind(SportEventInterface::class, SportEventRepository::class);
+//       $this->app->bind(SportEventInterface::class, SportEventAPIRepo::class);
```

Comment:
As part of the refactoring process I'd suggest that you control the binding between APIRepo and Repository via environment variable and config instead of commenting/uncommenting of the bindings.
Example:
```suggestion
if(Config::get('api.useApi')) {
       $this->app->bind(SportEventInterface::class, SportEventAPIRepo::class);
}
```